src/components/shared/forms/styles/btn.scss

```diff
@@ -34,24 +36,76 @@ $btnIconSize: 14px;
   }
 
   @include themify($themes) {
-    color: themed('colorHighest');
+    color: themed('colorBtnText');
+    background-color: themed('colorBtnBackground');
+    .btn-icon svg { fill: themed('colorBtnSvgFill'); }
 
-    .btn-icon svg { fill: themed('colorHighest'); }
+    &:focus {
+      border: 1px themed('colorBtnBorderFocus') dashed;
+    }
 
     &:hover {
-      color: themed('colorLowestSecond');
-      background-color: themed('colorLightestFirst');
+      color: themed('colorBtnTextHover');
+      background-color: themed('colorBtnBackgroundHover');
+
+      .btn-icon svg { fill: themed('colorBtnSvgFillHover'); }
+    }
+
+    &:active {
+      color: themed('colorBtnTextActive');
+      background-color: themed('colorBtnBackgroundActive');
+      border: 1px transparent dashed;
```

Comment:
If the property isn't changing with the theme, it shouldn't be inside the themify.